.env

```diff
@@ -1 +1,2 @@
 OPERATOR_SDK_VERSION=v0.18.2
+REVIEWERS=pdhanoyasplunk,smohan-splunk,kashok-splunk,akondur,sgontla
```

Comment:
Not sure how this works, if it needs, then we can add all the team?

Reply:
Yeah we can add whole team as well will need to include the all the usernames. I can do that if it's needed.